ts/src/binance.ts

```diff
@@ -747,6 +746,7 @@ export default class binance extends Exchange {
                         'income/asyn/id': 5,
                         'pmExchangeInfo': 0.5, // Weight(IP): 5 => cost = 0.1 * 5 = 0.5
                         'pmAccountInfo': 0.5, // Weight(IP): 5 => cost = 0.1 * 5 = 0.5
+                        'historicalTrades': 20,
```

Comment:
this endpoint in dapi, is private, not public:
https://dapi.binance.com/dapi/v1/historicalTrades

so, i've moved it from `dapiPublic -> dapiPrivate`